jest.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
     ],
     "^.+\\.(t|j)sx?$": ["@swc/jest", swcConfig],
   },
-  transformIgnorePatterns: ["node_modules/(?!(@runcitadel))/"],
+  transformIgnorePatterns: ["node_modules/(?!(dexie))/"],
```

Comment:
why was dexie added here?

Reply:
Because Jest is failing: https://github.com/getAlby/lightning-browser-extension/pull/747/files#r842242790 [same reason but dexie ig]

Not exactly sure why.
@escapedcat knows: https://github.com/getAlby/lightning-browser-extension/issues/1314#issue-1345365331

> **Unit-test setup**
>
> Looks like some unit-tests fail after an update to 16/18.
>
> This helps partly:
>- Updating runcitadel https://github.com/getAlby/lightning-browser-extension/pull/1280
>- `transformIgnorePatterns: ["node_modules/(?!(dexie))/"],`
> ...

Reply:
From jest:

```sh
    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation
```